Barotrauma/BarotraumaClient/ClientSource/Serialization/SerializableEntityEditor.cs

```diff
@@ -523,58 +523,62 @@ public GUIComponent CreateNewField(SerializableProperty property, ISerializableE
+                    };
+                    return valueField;
+                }
+                else
                 {
-                    Font = GUIStyle.SmallFont,
-                    Enabled = !Readonly,
-                    Selected = value,
-                    ToolTip = toolTip,
-                    OnSelected = (tickBox) =>
+                    GUITickBox propertyTickBox = new GUITickBox(new RectTransform(new Point(Rect.Width, elementHeight), layoutGroup.RectTransform, isFixedSize: true), displayName)
                     {
-                        if (SetPropertyValue(property, entity, tickBox.Selected))
-                        {
-                            TrySendNetworkUpdate(entity, property);
-                        }
-                        // Ensure that the values stay in sync (could be that we force the value in the property accessor).
-                        bool propertyValue = (bool)property.GetValue(entity);
-                        if (tickBox.Selected != propertyValue)
+                        Font = GUIStyle.SmallFont,
+                        Enabled = !Readonly,
+                        Selected = value,
+                        ToolTip = toolTip,
+                        OnSelected = (tickBox) =>
                         {
-                            tickBox.Selected = propertyValue;
-                            tickBox.Flash(Color.Red);
+                            if (SetPropertyValue(property, entity, tickBox.Selected))
+                            {
+                                TrySendNetworkUpdate(entity, property);
+                            }
+                            // Ensure that the values stay in sync (could be that we force the value in the property accessor).
+                            bool propertyValue = (bool)property.GetValue(entity);
+                            if (tickBox.Selected != propertyValue)
+                            {
+                                tickBox.Selected = propertyValue;
+                                tickBox.Flash(Color.Red);
+                            }
+                            return true;
                         }
-                        return true;
-                    }
-                };
-                refresh += () =>
-                {
-                    propertyTickBox.Selected = (bool)property.GetValue(entity);
-                };
-                if (!Fields.ContainsKey(property.Name)) { Fields.Add(property.Name.ToIdentifier(), new GUIComponent[] { propertyTickBox }); }
-                return propertyTickBox;
+                    };
+                    refresh += () =>
+                    {
+                        propertyTickBox.Selected = (bool)property.GetValue(entity);
+                    };
+                    if (!Fields.ContainsKey(property.Name)) { Fields.Add(property.Name.ToIdentifier(), new GUIComponent[] { propertyTickBox }); }
+                    return propertyTickBox;
+                }
```

Comment:
Unclear why this method was modified (seems unrelated to the rest of the changes in the PR) or what was actually modified about it.

Reply:
Odd. I've manually compared the files between the main branch and my branch and there are no changes here, so it may be an issue with Git?